lib/DDG/Spice/Untappd.pm

```diff
@@ -0,0 +1,38 @@
+package DDG::Spice::Untappd;
+
+# ABSTRACT: Discover your favorite beers
+
+use DDG::Spice;
+use Text::Trim;
+
+spice is_cached => 1;
+spice proxy_cache_valid => "200 1d";
+
+spice wrap_jsonp_callback => 1;
+
+spice to => 'https://api.untappd.com/v4/search/beer?' .
+                'client_id={{ENV{DDG_SPICE_UNTAPPD_CLIENT_ID}}}&' .
+                'client_secret={{ENV{DDG_SPICE_UNTAPPD_CLIENT_SECRET}}}&' .
+                'q=$1';
+
+triggers any => 'untappd';
+triggers end => qw/
+    ale apa ipa aipa dipa bipa porter stout bitter barleywine
+    saison lambik dubbel tripel quadrupel framboise gueuze kriek witbier
+    altbier weissbier dunkelweizen gose
+    hefeweizen kristalweizen roggenbier weizenbock
+    weissbier marzen dunkel weizenbier
+    bock doppelbock rauchbier schwarzbier
+    maibock eisbock kellerbier kolsch zoigl
+    lager pilsner
+    happoshu
+    sahti
+/;
+
+handle query_lc => sub {
```

Comment:
I have added a check for queries that contain only a trigger word.

I don't think skipping queries starting with "what" or "is" reduces false triggering. For queries like "What is india pale ale", untappd will return an empty list of results and user won't see this IA. And for queries like "what she's having stout" and "what ipa", we can find relevant results like "I'll Have What She's Having" or "What He's Having IPA".